vizro-core/docs/pages/explanation/contributing.md

```diff
@@ -1,192 +1,159 @@
+* [GitHub Codespaces](https://docs.github.com/en/codespaces). This is the recommended method if you are a new contributor and is the quickest and easiest way to get started. All development can be done in your browser in a temporary environment; you do not need to set up anything on your computer. Read [Develop on GitHub Codespaces](#develop-on-github-codespaces) for full instructions on how to do this.
+* Local machine. If you are more experienced then you might prefer to develop on your own computer. Read [Develop locally](#develop-locally) for full instructions on how to do this.
 
-## Want to request a feature?
+!!! note
 
-To raise a feature request, head to our [issues page](https://github.com/mckinsey/vizro/issues) and raise a ticket in the category `feature request`. We would appreciate if you searched the existing issues for a similar description before raising a new ticket. The team will then try to understand the request in more detail, explore the feasibility and prioritize it in relation to the current roadmap. We will get back to you as soon as possible with an estimate of whether and when this feature could be released.
+    For either method, Hatch is the _only development dependency_. You do not need to manually install Python or create any virtual environments to develop Vizro; all this will be handled for you behind the scenes by Hatch. We have also configured our codespace to pre-install Hatch. If you develop on GitHub Codespaces you don't need to install anything at all!
 
-## Got a Vizro question?
 
-We are happy to receive general questions around Vizro. Take a look at our [issues page](https://github.com/mckinsey/vizro/issues) and raise a ticket in the category `general question`. We would be grateful if you could check for any similar descriptions in the existing issues before opening a new ticket.
+## Develop on GitHub Codespaces
 
-If you are already active on the Plotly Community Forum, you can also ask your question over there in the [Dash Python category](https://community.plotly.com/c/python/25).
+!!! note
 
-## How to interact with the repository
+    There is no need to manually create a fork of the Vizro code if you use GitHub Codespaces, because fork is [automatically created for you](https://docs.github.com/en/codespaces/developing-in-a-codespace/using-source-control-in-your-codespace#about-automatic-forking).
+    
+To develop on [GitHub Codespaces](https://docs.github.com/en/codespaces), follow the below steps:
 
-The easiest way to get up and running is to [open the repository in GitHub Codespaces](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=626855142). This creates a temporary development environment with all the necessary configurations, making it especially convenient for tasks like reviewing pull requests.
+1. Use [Vizro's GitHub issues](https://github.com/mckinsey/vizro/issues) to discuss your contribution.
```

Comment:
Wondering if this should be pulled out to the top to explain clearly how to find a ticket to work on and what to do before you get started if the work is more than a small tweak?

Reply:
Maybe it's as simple as putting this big back that you removed (with some edit to add a bit about the requirements of a PR, which I've commented just below here)?

> Contributions of all experience levels are welcome! There are many ways to contribute, and we appreciate any help. Use our [issues page](https://github.com/mckinsey/vizro/issues) to discuss any contributions. Before opening a pull request, ensure you've first opened an issue to discuss the contribution.

Reply:
Also, I think it would help to edit slightly, here's my suggestion for the first sentence:

```suggestion
Contributions of all experience levels are welcome! There are many ways to contribute, and we appreciate any help: it doesn't have to be a pull request on our code. You can also raise an issue to suggest an improvement to Vizro or our documentation, request a feature, or ask and answer community questions. See the FAQ for guidance on how to do these things.

<Then continue about the issues page blah blah>
```

Reply:
@stichbury Do we use punctuation at the end of itemized lists?

Reply:
It depends. Typically we do if the bullets comprise one or more full sentence, but not if they're not because they're just statements. So it's like this pair of examples...

Some of the features include:
* feature 1
* feature 2
* feature 3

You need to consider the following:
* Do you really want to hurt me?
* Do you really want to make me cry?
* Do you know any other song lyrics that are questions?

(but it doesn't need to be a question, could just be a period as long as the text is a full and complete sentence).


HOWEVER, I don't always adhere to this and it can be difficult because sometimes bullets are not always consistent in the list itself (which could make it look weird) and sometimes there's another bulleted list nearby on the page that should be punctuated so it looks odd if there's one that is and one that isn't. 

TL;DR: I'd typically say, go with punctuation more times than not, and if in doubt, add it.

Reply:
This is the sort of guide I was trying to follow so good to know, thanks @stichbury!

And good suggestions with these edits, I will definitely change as you suggest.